api-reference/beta/resources/directory.md

```diff
@@ -0,0 +1,57 @@
+# directory resource type
+
+Represents deleted items. Deleted items are commonly referred to as recycle bin items.
+
+Actions for the directory resource include the ability restore or permanently delete an item. Currently, the restore and permanent delete functionality is available for users and groups. The following scenarios are enabled for deleted items:
```

Comment:
I'd re-write this - something like:

Deleted items are only currently supported for the groups resource type.  When a user or an administrator deletes a group, the group gets added to the deleted items "container".  There are a few operations that you can perform on deleted items:

- You can view all deleted (group) items using a GET method.  
- If a group was accidentally deleted, the group can be fully restored (with memberships and data fully intact) from the deleted items container. 
- It is also possible to permanently delete the deleted item using a DELETE method.  You should be very certain that you need to perform this operation;  once permanently deleted, there is no opportunity to restore.

Deleted items will remain in the deleted items container for 30 days, at which point they will be permanently deleted.